packages/flame/lib/src/widgets/animation_widget.dart

```diff
@@ -36,7 +39,8 @@ class SpriteAnimationWidget extends StatelessWidget {
   })  : _animationFuture = animation,
         _animationTicker = animationTicker,
         errorBuilder = null,
-        loadingBuilder = null;
+        loadingBuilder = null,
+        _onComplete = null;
```

Comment:
Why not expose onComplete from the default constructor too?

Reply:
The default constructor receives a concrete instance of a `SpriteAnimation`, which can have the callback in it. So I believe we don't need that callback there?

But at the same time, I agree that there is no harm in having it there as well

Reply:
True, I'm fine with either way then

Reply:
I would keep as is then :D